TestMempoolProgressAfterCreateEmptyBlocksInterval sometimes fails

<!--
Please fill in as much of the template below as you can.

Be ready for followup questions, and please respond in a timely
manner. We might ask you to provide additional logs and data (tendermint & app).
-->

**Tendermint version** (use `tendermint version` or `git rev-parse --verify HEAD` if installed from source):

a18380901fc8fd22696ceed27eb8cab6fd130610

**ABCI app** (name for built-in, URL for self-written if it's publicly available):

**Environment**:
- **OS** (e.g. from /etc/os-release):
- **Install tools**:
- **Others**:


**What happened**:

```
$ go1.13.5 test -run=TestMempoolProgressAfterCreateEmptyBlocksInterval -count=1000
--- FAIL: TestMempoolProgressAfterCreateEmptyBlocksInterval (0.27s)
panic: Timeout expired while waiting for new activity on the channel [recovered]
    panic: Timeout expired while waiting for new activity on the channel

goroutine 174149 [running]:
testing.tRunner.func1(0xc0542e8900)
    /home/cuonglm/sdk/go1.13.5/src/testing/testing.go:874 +0x3a3
panic(0x17ef2e0, 0x1b56530)
    /home/cuonglm/sdk/go1.13.5/src/runtime/panic.go:679 +0x1b2
github.com/tendermint/tendermint/consensus.ensureNewEventOnChannel(0xc03a41c6c0)
    /home/cuonglm/go/src/github.com/tendermint/tendermint/consensus/common_test.go:603 +0xec
github.com/tendermint/tendermint/consensus.TestMempoolProgressAfterCreateEmptyBlocksInterval(0xc0542e8900)
    /home/cuonglm/go/src/github.com/tendermint/tendermint/consensus/mempool_test.go:57 +0x2e9
testing.tRunner(0xc0542e8900, 0x1a874c8)
    /home/cuonglm/sdk/go1.13.5/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
    /home/cuonglm/sdk/go1.13.5/src/testing/testing.go:960 +0x350
exit status 2
FAIL    github.com/tendermint/tendermint/consensus  65.016s
```

**What you expected to happen**:

Test success.

**Have you tried the latest version**: yes/no

**How to reproduce it** (as minimally and precisely as possible):

See above command.

**Logs (paste a small part showing an error (< 10 lines) or link a pastebin, gist, etc. containing more of the log file)**:

**Config (you can paste only the changes you've made)**:

**node command runtime flags**:

**`/dump_consensus_state` output for consensus bugs**

**Anything else we need to know**:

The test seems very flaky, sometimes it's expired, sometimes not:

```
$ go1.13.6 test -run=TestMempoolProgressAfterCreateEmptyBlocksInterval -count=1000
--- FAIL: TestMempoolProgressAfterCreateEmptyBlocksInterval (0.10s)
panic: We should be stuck waiting, not receiving new event on the channel [recovered]
	panic: We should be stuck waiting, not receiving new event on the channel
```

--- a/consensus/common_test.go
+++ b/consensus/common_test.go
@@ -50,7 +50,7 @@ type cleanupFunc func()
 var (
 	config                *cfg.Config // NOTE: must be reset for each _test.go file
 	consensusReplayConfig *cfg.Config
-	ensureTimeout         = time.Millisecond * 100
+	ensureTimeout         = time.Millisecond * 200
 )
 
 func ensureDir(dir string, mode os.FileMode) {

--- a/consensus/mempool_test.go
+++ b/consensus/mempool_test.go
@@ -45,13 +45,15 @@ func TestMempoolNoProgressUntilTxsAvailable(t *testing.T) {
 func TestMempoolProgressAfterCreateEmptyBlocksInterval(t *testing.T) {
 	config := ResetConfig("consensus_mempool_txs_available_test")
 	defer os.RemoveAll(config.RootDir)
+
 	config.Consensus.CreateEmptyBlocksInterval = ensureTimeout
 	state, privVals := randGenesisState(1, false, 10)
 	cs := newStateWithConfig(config, state, privVals[0], NewCounterApplication())
+
 	assertMempool(cs.txNotifier).EnableTxsAvailable()
-	height, round := cs.Height, cs.Round
+
 	newBlockCh := subscribe(cs.eventBus, types.EventQueryNewBlock)
-	startTestRound(cs, height, round)
+	startTestRound(cs, cs.Height, cs.Round)
 
 	ensureNewEventOnChannel(newBlockCh)   // first block gets committed
 	ensureNoNewEventOnChannel(newBlockCh) // then we dont make a block ...

